changelog/Microsoft.DirectoryServices.json

```
@@ -36,6 +36,24 @@
       "WorkloadArea": "Applications",
       "SubArea": ""
     },
+    {
+      "ChangeList": [
+        { 
+          "Id": "daa9406f-39d6-48fc-aaa4-452a7494b05c",
+          "ApiChange": "Method",
+          "ChangedApiName": "assignedPrincipals",
+          "ChangeType": "Addition",
+          "Description": "Added the [assignedPrincipals](https://docs.microsoft.com/en-us/graph/api/unifiedroledefinition-assignedprincipals) method to the [unifiedRoleDefinition](https://docs.microsoft.com/en-us/graph/api/resources/unifiedRoleDefinition?view=graph-rest-beta) resource.",
```

Suggestion to enhance changelog description so customer know the benefit of the API before they can get into the docs.
```suggestion
          "Description": "Added the [assignedPrincipals](https://docs.microsoft.com/en-us/graph/api/unifiedroledefinition-assignedprincipals) method to the [unifiedRoleDefinition](https://docs.microsoft.com/en-us/graph/api/resources/unifiedRoleDefinition?view=graph-rest-beta) resource. Use this API to retrieve the principals (users, groups, and service principals) directly or transitively assigned to a specific role for different scopes.",
```